docs/production/setting-up-javascript.md

```diff
@@ -59,11 +59,13 @@ Rather than using `initAll`, you can initialise individual components by identif
 
 ```html
 <script>
-var PasswordReveal = window.MOJFrontend.PasswordReveal
-var $passwordReveal = document.querySelector('[data-module="moj-password-reveal"]')
-if ($passwordReveal) {
+  var PasswordReveal = window.MOJFrontend.PasswordReveal
```

Comment:
When the time's right, there's a lovely ESLint plugin for Markdown:

* https://github.com/alphagov/govuk-frontend/pull/3698

Reply:
👍 I'll do this in a follow on PR